help/en/html/doc/Technical/ReleaseProcess.md

```diff
@@ -20,21 +20,28 @@ _Before_ a release, the development installers will be named with the branch bei
 
 Each PR that's merged for inclusion can require an increment of the first, second or third digit.  For this to work, we need a very reliable way to identify the right one, and make sure it takes effect.
 
- - We'll define three new labels for GitHub PRs:  Major Change, Minor Change, Update Only
+ - We'll define four new labels for GitHub PRs:
+   - _Breaking Change_ - requires a major version change because it breaks outside code
+   - _Feature_ - changes the visible API
+   - _Fix_ - no change to visible API, but triggers a version change absent other non-chore changes
+   - _Chore_ - no change to Java code or published artifacts (e.g. updates build scripts and CI processes), does not trigger version change
  - A PR must have exactly one of those applied for it to be merged
  - The author of the PR can propose a label, in which case only one reviewer is required.
  - If the author of the PR does not propose a label, two reviewers must sign off on the right label.
```

Comment:
GitHub can enforce that a minimum number of accepting reviews be required to allow a PR to be merged, but that number cannot be made conditional upon labels.

Reply:
I'm told, by one of my grad students who hasn't actually done it, that a GitHub action can enforce this.  She thinks it would be "just a smarter WIP action".